contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -91,6 +91,14 @@ static ObjectAddress get_trigger_object_address(List *object, Relation *relp, bo
 Oid			get_tsql_trigger_oid(List *object, const char *tsql_trigger_name, bool object_from_input);
 static Node *transform_like_in_add_constraint(Node *node);
 
+/*****************************************
+ * 			Protocol Hooks
+ *****************************************/
+
+IsTdsClientHookType PrevIsTdsClientHook = NULL;
```

Comment:
We cannot have such a hook in opensource engine, please consult a senior dev like @kuntalghosh @surendravishnoi

Reply:
This hook was added at the suggestion of Surendra. I will contact Kuntal to get his opinion as well.

Reply:
Reworked to use a rendezvous variable, no longer need the hook